--- postfix-1.1.5/src/local/maildir.c.orig	Wed Nov 28 00:43:47 2001
+++ postfix-1.1.5/src/local/maildir.c	Thu Mar 21 15:04:26 2002
@@ -139,7 +139,7 @@
 	vstring_sprintf(why, "create %s: %m", tmpfile);
     } else {
 	if ((mail_copy_status = mail_copy(COPY_ATTR(state.msg_attr),
-					dst, copy_flags, "\n", why)) == 0) {
+					dst, copy_flags, "\r\n", why)) == 0) {
 	    if (sane_link(tmpfile, newfile) < 0
 		&& (errno != ENOENT
 		    || (make_dirs(curdir, 0700), make_dirs(newdir, 0700)) < 0
